db/seeds.rb

```diff
@@ -1,4 +1,31 @@
+User.create!(name: "Example User",
+            email: "example@railstutorial.org",
+            profile: "Nice to meet you",
+            password: "foobar",
+            password_confirmation: "foobar" )
+
+10.times do |n|
+  name  = Faker::Name.name
+  email = "example-#{n+1}@railstutorial.org"
+  profile = Faker::ProgrammingLanguage.name
+  password = "password"
+  User.create!(name:  name,
+               email: email,
+               profile: profile,
+               password: password,
+               password_confirmation: password)
```

Comment:
スペースが１個多いのが気になるｗ

あとモダンな書き方にするならこうかな。
```suggestion
  User.create!(name:,
               email:,
               profile:,
               password:,
               password_confirmation: password)
```